src/connectors/hits/__tests__/connectHits-test.ts

```diff
@@ -145,37 +167,47 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/hits/js/#co
 
     const hits = [
       {
+        objectID: '1',
         _highlightResult: {
           foobar: {
             value: `<script>${TAG_PLACEHOLDER.highlightPreTag}foobar${TAG_PLACEHOLDER.highlightPostTag}</script>`,
+            matchLevel: 'none',
+            matchedWords: ['foobar'],
           },
         },
       },
     ];
 
-    const results = new SearchResults(helper.state, [{ hits }]);
-    widget.render({
-      results,
-      state: helper.state,
-      helper,
-      createURL: () => '#',
-    });
+    const results = new SearchResults(helper.state, [
+      createSingleSearchResponse({ hits }),
+    ]);
+    widget.render!(
+      createRenderOptions({
+        results,
+        state: helper.state,
+        helper,
+      })
+    );
 
-    const escapedHits = [
+    // @ts-ignore
+    const expectedHits: EscapedHits = [
```

Comment:
We should be able to remove the `@ts-ignore`, cast and `__escaped` by using:

```ts
const expectedHits = escapeHits([ /* ... */ ]);
```

Reply:
The test here is asserting that we proper escape hits behind the scenes, so the `expectedHits` contains escaped content already. I am doing a ts ignore because i don't know how to assign an array with a property in javascript. You do?